pkg/fs/fs.go

```diff
@@ -138,14 +137,19 @@ func copyDir(src, dst string, info os.FileInfo) error {
 		return err
 	}
 	// copy every source dir entry
-	entries, err := ioutil.ReadDir(src)
+	entries, err := os.ReadDir(src)
 	if err != nil {
 		return err
 	}
 	for _, entry := range entries {
 		entrySrc := filepath.Join(src, entry.Name())
 		entryDst := filepath.Join(dst, entry.Name())
-		if err := copy(entrySrc, entryDst, entry); err != nil {
+		info, err := entry.Info()
+		if err != nil {
+			return err
+		}
+
+		if err := copy(entrySrc, entryDst, info); err != nil {
```

Comment:
this seems the only change that is not a 1to1 replacement :+1:

Reply:
Yes, `os.ReadDir` returns a slice of `os.DirEntry` rather than a slice of `fs.FileInfo`. This is the only function that is not a direct replacement.